src/libraries/Microsoft.Bcl.Numerics/src/Microsoft.Bcl.Numerics.csproj

```
@@ -0,0 +1,29 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>$(NetCoreAppCurrent);netstandard2.0;netstandard2.1;$(NetFrameworkMinimum)</TargetFrameworks>
```

Why do you need a `NetCoreAppCurrent` TFM for htis package? The netstandard2.1 TFM should be sufficient as a type-forward. Same as Microsoft.Bcl.AsyncInterfaces.

---

So I spent some time thinking about this. You want to remove the NetCoreAppCurrent TFM (and re-order TFMs - netstandard2.1 only has type forwards and hence should be last in the list so that VS doesn't pick it as the default TFM):

```suggestion
    <TargetFrameworks>netstandard2.0;$(NetFrameworkMinimum);netstandard2.1</TargetFrameworks>
```